packages/embark/src/lib/modules/solidity/solcW.js

```diff
@@ -1,6 +1,6 @@
+import { ProcessLauncher } from 'embark-core';
 import {joinPath} from 'embark-utils';
 let currentSolcVersion = require('../../../../package.json').dependencies.solc;
```

Comment:
Since this could be a range instead of a version should we use semver (later portion of this script) to test the version available satisfies the range as opposed to being `===`?

Reply:
@michaelsbradleyjr That's a good question. I think Solidity isn't following semver though (e.g. breaking changes between x.4.y and x.5.y), which might be the reason this script checks for exact match. 

Either way, that'd be done in a separate commit, this one just moves stuff around.